ui/components/multichain/import-tokens-modal/import-tokens-modal-confirm.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  ButtonPrimary,
+  ButtonSecondary,
+  Text,
+} from '../../component-library';
+import {
+  AlignItems,
+  Display,
+  Size,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import TokenBalance from '../../ui/token-balance/token-balance';
+import Identicon from '../../ui/identicon';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getPendingTokens } from '../../../ducks/metamask/metamask';
+
+export const ImportTokensModalConfirm = ({ onBackClick, onImportClick }) => {
+  const t = useI18nContext();
+  const pendingTokens = useSelector(getPendingTokens);
+
+  return (
+    <Box padding={4}>
```

Comment:
We need to change it to only padding top and bottom. Since, paddingLeft and paddingRight is extra padding to the box
```suggestion
    <Box paddingTop={4} paddingBottom={4}>
```
![Screenshot 2023-08-02 at 5 43 10 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/b9f7702b-7dab-43d1-a890-072f2035c333)

Reply:
Set!